core/src/scene/sceneLoader.cpp

```diff
@@ -64,6 +64,56 @@ void printFilters(const SceneLayer& layer, int indent){
     }
 };
 
+void SceneLoader::applyGlobalProperties(Node& node, Scene& scene) {
+    switch(node.Type()) {
+    case NodeType::Scalar:
+        {
+            std::string key = node.Scalar();
+            if (key.compare(0, 7, "global.") == 0) {
+                key.replace(0, 7, "");
+                key = std::regex_replace(key, std::regex("\\."), DELIMITER);
+                node = scene.globals()[key];
+            }
+        }
+        break;
+    case NodeType::Sequence:
+        for (const auto& n : node) {
+            Node nextNode = n;
+            applyGlobalProperties(nextNode, scene);
+        }
+        break;
+    case NodeType::Map:
+        for (const auto& n : node) {
+            Node nextNode = n.second;
+            applyGlobalProperties(nextNode, scene);
+        }
+        break;
+    default:
+        break;
+    }
+}
+
+void SceneLoader::parseGlobals(const Node& node, Scene& scene, const std::string& key) {
+    switch (node.Type()) {
+    case NodeType::Scalar:
+    case NodeType::Sequence:
+        {
+            auto& g = scene.globals();
+            g[key] = YAML::Clone(node);
```

Comment:
I don't think `Clone` is necessary here, it's more efficient to assign the node directly.